docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -17,3 +17,72 @@ on the default backend.
         commands:
             - python "${helloworld}"
 ```
+
+## Custom memory limit
+
+When jobs exceeds the memory limits, you will get out of memory (OOM) error and job gets [`OOMKilled`](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits). With REANA, you can configure cluster wide **memory** limits to make the efficient use of available resources. Note that you can also decrease the memory limit to increase the chances of jobs being scheduled earlier.
+
+To set the memory limit of a job you must specify `kubernetes_memory_limit` in _**every step specification**_.
```

Comment:
I wouldn't say **every** as it implies that you're force to set it for all the steps.

```
To set the memory limit of a job you can specify `kubernetes_memory_limit` in _**each step specification**_.
```